.github/workflows/release.yml

```diff
@@ -0,0 +1,63 @@
+name: Release
+on:
+  release:
+    types:
+      # works for building pre-releases
+      - published
```

Comment:
In https://github.com/newrelic/k8s-metadata-injection/pull/73 the schema to run is different.

We should converge on a single solution @roobre

Reply:
these 2 worked the best for me. If works if you do a pre-release, a release or change from pre-release to release

Reply:
@roobre should we update the other PR then?

Reply:
I think that `[ prereleased, released ]` was slightly more clear than `[ published, released ]`, as in the latter it is not clear whether prereleases are included (unless you add a comment). As far as I can tell, they are functionally identical.

However it's not a strong preference, my only slight plus is the readability thing.

Reply:
I think prereleased **also** triggers when you create a **draft** prerelease , not just when you really publish it, which then might force you to check if it's not a draft. 
Ideally **published** would be enough but it doesn't trigger when you change prelrelease to release, which it's nice to have

Reply:
> I think prereleased also triggers when you create a draft prerelease , not just when you really publish it, which then might force you to check if it's not a draft.

That would be bad indeed, let me check it